lampadaires/README.md

```diff
@@ -0,0 +1,5 @@
+## Lampadaires
+
+Use a personal javascript script to active/deactive a layer
+
+![QGIS logo](lampadaires.qgs.png)
```

Comment:
```suggestion
![QGIS logo](lampadaires.qgs.jpg)
```

The PNG file doesn't not exist.

Can you enable an markdown extension in your IDE ? It might raise a warning.
PyCharm does it